tool/teleport/common/teleport.go

```diff
@@ -1017,7 +1017,7 @@ func dumpConfigFile(outputURI, contents, comment string) (string, error) {
 func onSCP(scpFlags *scp.Flags) (err error) {
 	// when 'teleport scp' is executed, it cannot write logs to stderr (because
 	// they're automatically replayed by the scp client)
-	utils.SwitchLoggingToSyslog()
+	slog.SetDefault(slog.New(logutils.DiscardHandler{}))
```

Comment:
Is it a problem that these aren't equivalent anymore?

Reply:
No strong opinions on my part but attempting syslog first does seem better from a behavioral-compatibility point of view.

Reply:
Sounds OK.